elastic/elastic-package
# Implement terraform based system tests

~I'm opening this as draft to present the progress on the TF-based system tests.~

~Please do not consider this as final form, as I will modify it soon.~

Issue: https://github.com/elastic/elastic-package/issues/89

How to test this:

```
export AWS_ACCESS_KEY_ID="secret"
export AWS_SECRET_ACCESS_KEY="secret"

cd test/packages/aws
elastic-package test system --data-streams ec2_metrics -v
```

Tf definitions in: `data_stream/ec2_metrics/_dev/deploy/tf/main.tf` (or other .tf files)

EDIT:

I switched from SNS to EC2 metrics during tests as metrics tend to appear quicker.
